generators/client/templates/react/webpack/webpack.dev.js.ejs

```diff
@@ -79,8 +79,9 @@ module.exports = (options) => webpackMerge(commonConfig({ env: ENV }), {
         '/h2-console',
         '/auth'
       ],
-      target: 'http://127.0.0.1:<%= serverPort %>',
```

Comment:
Not sure you should change the server port here, it's a parameter because people will probably want to change this for microservices.

Reply:
Sorry it was a mistake, I have fixed it.